compendium/DeclarativeServices/src/manager/ComponentConfigurationImpl.cpp

```diff
@@ -255,4 +255,10 @@ InstanceContextPair ComponentConfigurationImpl::CreateAndActivateComponentInstan
   return std::make_pair(componentInstance, ctxt);
 }
 
+void ComponentConfigurationImpl::UnbindAndDeactivateComponentInstanceHelper(std::shared_ptr<ComponentInstance> componentInstance)
```

Comment:
does this have to be passed by value? It seems like this should be passed by ref